examples/plotting/lasco_overlay.py

```diff
@@ -0,0 +1,61 @@
+"""
+==============================================
+Overlay an AIA image on a LASCO C2 coronagraph
+==============================================
+
+This tutorial shows the steps needed to overlay the disc and off-limb
+components of an AIA image within the masked occulter of a LASCO C2 image.
+It also demonstrates the usage of hypy to download a JPEG 2000 image from
+HelioViewer.
+"""
+from datetime import datetime
+
+import hvpy
+import matplotlib.pyplot as plt
+from hvpy.datasource import DataSource
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.data.sample
+from sunpy.coordinates import Helioprojective
+from sunpy.map import Map
+
+###############################################################################
+# First, specify the observation date and time as a datetime object that will
+# be passed to hvpy. The downloaded JPEG 2000 image is saved and then loaded
+# into Map object.
+
+dt = datetime(2011, 6, 7, 6, 34)
+
+jp2 = hvpy.save_file(hvpy.getJP2Image(dt, DataSource.LASCO_C2.value),
+                     filename="lasco.jp2", overwrite=True)
+lasco = Map(jp2)
+aia = Map(sunpy.data.sample.AIA_171_IMAGE)
```

Comment:
```suggestion

# hvpy uses the standard datetime instead of astropy.time.
lasco_jp2_file = hvpy.save_file(hvpy.getJP2Image(datetime(2011, 6, 7, 6, 34), DataSource.LASCO_C2.value),
                     filename="lasco.jp2", overwrite=True)
lasco_map = Map(lasco_jp2_file)
aia_map = Map(sunpy.data.sample.AIA_171_IMAGE)
```